lib/src/linking.rs

```diff
@@ -90,7 +140,7 @@ impl ComponentCtx {
             wasi: builder.build(),
             session,
             guest_profiler: guest_profiler.map(Box::new),
-            limiter: Limiter::default(),
+            limiter: Limiter::new(100, 100),
```

Comment:
We'll need to change this, but I'm not too worried about it now as we're not officially supporting components yet.